source/content/cheatsheet/index.md

```diff
@@ -0,0 +1,917 @@
+# Cheat sheet
```

Comment:
Especially in Chapter/Section headings, we are currently capitalizing all proper nouns.